Feature/uno 93

Bassani-Paulo opened the pull request:
Criei a lógica do actionswipe e arrumei algumas cagadas do fernando

Bassani-Paulo committed:
arrumei as letras maiusculas do fer

Bassani-Paulo committed:
Merge branch 'develop' into feature/UNO-93

Bassani-Paulo committed:
Merge branch 'develop' into feature/UNO-93

Bassani-Paulo committed:
detalhes finais

Bassani-Paulo committed:
arrumando o enum

Fercomp commented on WhyNot/WhyNot/Scenes/Experiences/ExperiencesView.swift:
```diff
@@ -1 +1,42 @@
+
+import SwiftUI
+
+var getLikedStateFromExp = false
```
Variávell global, poderia ser substituída por um @State dentro da struct

Bassani-Paulo replied:
isso é um mock, o verdadeiro booleano vai ser tirado da experiencia

Fercomp commented on WhyNot/WhyNot/Scenes/Experiences/ExperiencesView.swift:
```diff
@@ -1 +1,42 @@
+struct ExperiencesView: View {
+    var body: some View {
+        List {
+                    Text("cardCell")
```
Indentação do List está bizarra

Bassani-Paulo committed:
ajustes

Quando eu peguei a branch inicialmente, o TitleColor do Assets tava sem o WN então tirei pra pegar o certo, mas corrigiram isso no assets e eu só vi agora, então devolvi o WN

Bassani-Paulo committed:
Merge branch 'develop' into feature/UNO-93

Bassani-Paulo committed:
arrumei os conflitos

Bassani-Paulo committed:
ajuste de tradução

luizgustavo09 reviewed (approved):
LGTM!

LarIsCoding reviewed (commented):
onde está a lógica do swipe? 🤔
